exercises/practice/dnd-character/tests/Tests.elm

```diff
@@ -0,0 +1,167 @@
+module Tests exposing (tests)
+
+import Dict
+import DndCharacter exposing (Character)
+import Expect
+import Random exposing (Generator, Seed)
+import Set
+import Test exposing (Test, describe, skip, test)
+
+
+generateValuesFromSeed : Int -> Generator a -> Seed -> List a
```

Comment:
I would probably hard code ability and character versions of this, to save computation time and make the tests a tiny bit less complicated.